public/wp-content/themes/glific/header.php

```diff
@@ -73,4 +80,4 @@
 	</div>
 </header>
 
-<body>
+<body class="<?php echo !empty($banner_content) ? 'mt-28.5' : 'mt-20.5' ; ?> " >
```

Comment:
Possible to use Laravel blade format instead of using raw PHP?

Reply:
Need to look into it.